app/index.js

```diff
@@ -22,25 +23,39 @@ process.chdir(__dirname);
 // Setup the app
 var port = process.env.PORT || 3000;
 app.listen(port);
-app.use(bodyParser.urlencoded({
-	extended: true
-}));
+app.use(
+  bodyParser.urlencoded({
+    extended: true
+  })
+);
 app.use(bodyParser.json());
 
 let spaces = 4;
-if(process.env.NODE_ENV === 'production') {
+if (process.env.NODE_ENV === 'production') {
   spaces = 0;
 }
 app.set('json spaces', spaces);
 
 // Set the version
 app.set('version', require('../package.json').version);
 
+const log = require('./helpers/logger');
+exec(`git rev-list -n1 HEAD`, (err, stdout, stderr) => {
```

Comment:
![giphy](https://user-images.githubusercontent.com/23438560/68703726-2ccbbf80-0559-11ea-8280-d3494b50047d.gif)

Reply:
This little snippet allows you to get the exact commit hash that was deployed (just in case someone deploys from a hash instead of a tag)